packages/bpk-component-graphic-promo/src/custom-breakpoints.module.scss

```diff
@@ -0,0 +1,74 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2022 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// FIXME: Replace the custom breakpoints with on es from Backpack when ready. (ST-4102)
+
+@function pxtorem($px) {
+  @return ($px / 16) * 1rem;
+}
+
+$breakpoint-landscape-desktop-min: pxtorem(801);
+$breakpoint-landscape-tablet-max: pxtorem(800);
+$breakpoint-landscape-tablet-min: pxtorem(701);
+$breakpoint-portrait-large-max: pxtorem(700);
+$breakpoint-portrait-large-min: pxtorem(513);
+$breakpoint-portrait-medium-max: pxtorem(512);
+$breakpoint-portrait-medium-min: pxtorem(425);
+$breakpoint-portrait-small-max: pxtorem(424);
+
+@mixin breakpoint-landscape-desktop {
+  @media (min-width: $breakpoint-landscape-desktop-min) {
+    @content;
+  }
+}
+
+@mixin breakpoint-landscape-tablet {
+  @media (min-width: $breakpoint-landscape-tablet-min) and (max-width: $breakpoint-landscape-tablet-max) {
```

Comment:
nitpick: Might be able to avoid specifying ranges which might reduce the variable usage here.

taken from: https://www.freecodecamp.org/news/the-100-correct-way-to-do-css-breakpoints-88d6a5ba1862/

```javascript
@mixin for-phone-only {
  @media (max-width: 599px) { @content; }
}
@mixin for-tablet-portrait-up {
  @media (min-width: 600px) { @content; }
}
@mixin for-tablet-landscape-up {
  @media (min-width: 900px) { @content; }
}
@mixin for-desktop-up {
  @media (min-width: 1200px) { @content; }
}
@mixin for-big-desktop-up {
  @media (min-width: 1800px) { @content; }
}
```

Reply:
Happy to leave it as-is if doing this approach would complicate things elsewhere